netbout-web/src/test/java/com/netbout/rest/MarkdownTxtmarkTest.java

```diff
@@ -0,0 +1,274 @@
+/**
+ * Copyright (c) 2009-2015, netbout.com
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are PROHIBITED without prior written permission from
+ * the author. This product may NOT be used anywhere and on any computer
+ * except the server platform of netbout Inc. located at www.netbout.com.
+ * Federal copyright law prohibits unauthorized reproduction by any means
+ * and imposes fines up to $25,000 for violation. If you received
+ * this code accidentally and without intent to use it, please report this
+ * incident to the author by email.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+ * ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT
+ * OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
+ * SUCH DAMAGE.
+ */
+package com.netbout.rest;
+
+import com.jcabi.matchers.XhtmlMatchers;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Ignore;
+import org.junit.Test;
+
+/**
+ * Test case for {@link MarkdownTxtmark}.
+ *
+ * @author Dmitry Zaytsev (dmitry.zaytsev@gmail.com)
+ * @version $Id$
+ * @since 2.23
+ * @todo #867:30min/DEV MarkdownTxtmark not implemented but has to be.
+ *  See #847 for details.
+ *  This class should be implementation of Markdown using TxtMark.
+ *  See https://github.com/rjeschke/txtmark.
+ *  After that don't forget remove Ignore annotation from unit tests.
+ *
+ */
+public final class MarkdownTxtmarkTest {
+    /**
+     * MarkdownTxtmark can handle whitespace after links.
+     * @throws Exception If there is some problem inside
+     */
+    @Test
+    @Ignore
+    public void handlesWhitespaceAfterLinks() throws Exception {
+        MatcherAssert.assertThat(
+            new MarkdownTxtmark().html(
+                "Hi [google](http://www.google.com) how are you?"
+            ),
+            Matchers.equalTo(
+                MarkdownTxtmarkTest.replaceNewLine(
+                    // @checkstyle LineLengthCheck (1 line)
+                    "<p>Hi \n<a href=\"http://www.google.com\">google</a> how are you?</p>\n"
```

Comment:
@longtimeago I just copy test conditions from MakdownPegdownTest 
to be sure that a new implementation will give same results